hack/build-repo.sh

```
@@ -0,0 +1,37 @@
+    continue
+  fi
+  mkdir -p "$TMP/$chart-$version"
+  (set -x; git archive "$commit" "$chart" | tar -xf- --strip-components=1 -C "$TMP/$chart-$version")
+
+  if [ -d "$TMP/$chart-$version/charts" ]; then
+    (set -x; find "$TMP/$chart-$version/charts" -type l -maxdepth 1 -mindepth 1) | awk -F/ '{print $NF}' | while read library_chart; do
+      (set -x; rm -f "$TMP/$chart-$version/charts/$library_chart")
+      (set -x; cd ../library && git archive "$commit" "$library_chart" | tar -xf- -C "$TMP/$chart-$version/charts/")
+    done
+  fi
+  rm -rf "$tmp"
```

Shellcheck (0.10.0)

[warning] tmp is referenced but not assigned.

(SC2154)